wiki/Beatmapping/Overall_difficulty/fr.md

```diff
@@ -17,19 +16,30 @@ L'**overall difficulty** (abrégé en ***OD***) définit la difficulté d'obteni
 
 ## Timing
 
-Des valeurs d'OD plus élevées signifient une hit window plus courte pour frapper les [objets](/wiki/Hit_object), à la fois en général et pour obtenir des [scores](/wiki/Gameplay/Score) élevés. La largeur de l'intervalle pour frapper une note, centrée sur le timing correct de l'objet, est définie par le tableau suivant :
+Des valeurs d'OD plus élevées signifient une fenêtre de hit plus courte pour frapper les [objets](/wiki/Hit_object), à la fois en général et pour obtenir des [scores](/wiki/Gameplay/Score) élevés. Le hit error maximale autorisée pour chaque objet dans osu! et osu!mania, centrée sur le timing correct de l'objet, est définie par les tableaux suivants :
```

Comment:
would "erreur de hit" also work?

Reply:
It sounds strange to me this formulation